blockchain-updates/build.sbt

```diff
@@ -0,0 +1,9 @@
+name := "blockchain-updates"
+
+libraryDependencies += "org.apache.kafka" %% "kafka" % "2.1.0"
```

Comment:
Kafka version should probably be fixed in `Dependencies.scala` (see `dependencyOverrides`)